Gem/Code/Source/Components/NetworkMatchComponent.cpp

```diff
@@ -160,13 +166,68 @@ namespace MultiplayerSample
                 {
                     highestCoins = aznumeric_cast<int>(state.m_score);
                     results.m_winningPlayerName = state.m_playerName;
```

Comment:
if we're always calling `FindWinner()` to populate `results.m_winningPlayerName`, maybe we should skip setting it here in case a bad result gets mistakenly held over in the future

Reply:
Oh, yeah, good point!